src/backends/perf_pt/mod.rs

```diff
@@ -624,12 +624,12 @@ mod tests {
         }
 
         // Make the trace and write it to a file.
-        let mut fh = NamedTempFile::new().unwrap();
-        trace.to_file(&mut fh);
-        fh.sync_all().unwrap();
+        let mut tmpf = NamedTempFile::new().unwrap();
+        trace.to_file(&mut tmpf.as_file_mut());
+        tmpf.as_file().sync_all().unwrap();
 
         // Check the resulting file makes sense.
-        let file = File::open(fh.path().to_str().unwrap()).unwrap();
```

Comment:
Similarly, this change would not be necessary if we left the variable name as-is.